### ISSUE TITLE
Ulnar nerve distribution to findings

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10723]
**Preferred term label:**
Ulnar nerve distribution to findings
Radial nerve distribution to findings

**Synonyms**

**Definition (free text, please give PubMed ID)**
for ulnar: Paresthesias in small finger, ulnar half of ring finger, and ulnar dorsal hand; weakness in adductor pollicis, deep head flexor pollicis brevis, interossei, and lumbricals 3 and 4; hypothenar atrophy

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/browse/term/HP:0012186 is related but a specific cause.  It would be good to have 

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
For ulnar:
Cubital tunnel syndrome
Ulnar tunnel syndrome

**Your nano-attribution (ORCID)**
